docs/md_docs/contributing.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to ansible-bender
```

Comment:
would be awesome to merge this with CONTRIBUTING.md so that we don't have two distinct contributing guides